src/shared/helpers/EventHelper.ts

```diff
@@ -14,6 +14,13 @@ import { OSModel } from '../../core/modelRepo/OSModel';
 import { SubscriptionModel } from '../../core/models/SubscriptionModels';
 
 export default class EventHelper {
+  // Sets the app badge count: https://developer.mozilla.org/en-US/docs/Web/API/Badging_API
+  static setAppBadgeCount(notification: OSNotificationDataPayload): void {
```

Comment:
This runs on the page, which means the badge would only get set if the end-user has a page open to the site. Shouldn't we read the value from the notification payload in the ServiceWorker and update it there? `navigator` is available in the Service Worker, is `setAppBadge` not?